p8e-api/build.gradle

```diff
@@ -157,3 +181,64 @@ dependencyManagement {
         cacheChangingModulesFor 0, 'seconds'
     }
 }
+
+task downloadCosmosProtos(type: Download) {
```

Comment:
maybe unclear to me, if we have gradle tasks to download protos, do we still need to commit the proto files below?

Reply:
that's how I initially had it set up, but changed it because of @scirner22 's comment above https://github.com/provenance-io/p8e/pull/9#discussion_r595658957 ... with this way you don't have to be actively connected to the internet to build (so you can work while on an ✈️ I suppose)... but maybe I can find a way to cache the zip/tar to behave kind of like the other dependencies while offline, use the cache if present and/or download fails